apps/agreement/contracts/Agreement.sol

```diff
@@ -559,7 +559,8 @@ contract Agreement is IArbitrable, IForwarder, AragonApp {
         // Create dispute
         address submitter = _action.submitter;
         require(feeToken.safeTransferFrom(submitter, address(this), missingFees), ERROR_ARBITRATOR_FEE_DEPOSIT_FAILED);
-        require(feeToken.safeApprove(recipient, totalFees), ERROR_ARBITRATOR_FEE_APPROVAL_FAILED);
+        _approveArbitratorFeeTokens(feeToken, recipient, 0);
```

Comment:
Should we just do the two approval calls inside the `_approveArbitratorFeeTokens` function? Also let's add a comment on why we are doing this

Reply:
Let's add a comment definitely, but note that this way we are reusing the call encode, just saying because of the size limit haha